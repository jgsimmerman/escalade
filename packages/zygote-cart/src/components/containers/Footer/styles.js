import css from 'styled-jsx/css';

export default css`
  .zygoteFooter {
    text-align: center;
    background-color: #f7f7f7;
    padding: 10px 20px;
    cursor: default;
  }
  .zygoteFooter span {
    font-size: 0.75em;
    text-transform: uppercase;
    font-weight: 700;
  }
  .zygoteFooter svg {
    width: 11px;
    height: 11px;
    margin-right: 1px;
  }
`;
