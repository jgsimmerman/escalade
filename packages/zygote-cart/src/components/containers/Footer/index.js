import React, { Component } from 'react';
import styles from './styles';

export default class Footer extends Component {
  render() {
    return (
      <div className="zygoteFooter">
        <span>{this.props.footerMessage}</span>
        <style jsx global>
          {styles}
        </style>
      </div>
    );
  }
}

Footer.defaultProps = {
  footerMessage: 'This is a custom footer message'
};
