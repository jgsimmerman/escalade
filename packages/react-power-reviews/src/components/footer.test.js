import React from "react"
import renderer from "react-test-renderer"

import Footer from "./footer"

describe(`Footer`, () =>
	it(`renders correctly`, () => {
		const container = renderer.create(<Footer />)
		const tree = container.toJSON()
		expect(tree).toMatchSnapshot()
	})
)