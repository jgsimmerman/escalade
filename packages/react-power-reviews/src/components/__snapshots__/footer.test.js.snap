// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="css-17qhf9j"
>
  <ul>
    <li>
      <Link
        to="/privacy-policy"
      >
        Privacy Policy
      </Link>
    </li>
    <li>
      <a
        href="/admin/"
      >
        CMS
      </a>
    </li>
  </ul>
</footer>
`;
